examples/bootstrap/.happy/config.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Should this have the other root fields? (`config_version`, `default_env`, `app`, etc)

Reply:
This file should have been committed, good catch.